src/misc/cpan-faq.html

```diff
@@ -1029,7 +1033,7 @@ <h3>
 </p>
 <p>
     Here is a checklist from <a href=
-    "http://perldoc.perl.org/perlbug.html">perlbug</a>, a bug reporting tool
+    "https://perldoc.perl.org/perlbug">perlbug</a>, a bug reporting tool
```

Comment:
Long term, would it be better to just remove this section from here and link directly to the list?  (i.e. reduce duplication)

Reply:
Possibly, though the best we could do is link to https://perldoc.perl.org/perlbug#DESCRIPTION - and frankly these docs very rarely change and are even less likely to do so now